src/core/Akka/Configuration/Pigeon.conf

```diff
@@ -315,7 +315,23 @@ akka {
 	  # This will be used if you have set "executor = "fork-join-executor""
       # Underlying thread pool implementation is scala.concurrent.forkjoin.ForkJoinPool
       fork-join-executor {
-       dedicated-thread-pool{ #settings for Helios.DedicatedThreadPool
+        # Min number of threads to cap factor-based parallelism number to
+        parallelism-min = 8
+
+        # The parallelism factor is used to determine thread pool size using the
+        # following formula: ceil(available processors * factor). Resulting size
+        # is then bounded by the parallelism-min and parallelism-max values.
+        parallelism-factor = 1.0
+
+        # Max number of threads to cap factor-based parallelism number to
+        parallelism-max = 64
+
+        # Setting to "FIFO" to use queue like peeking mode which "poll" or "LIFO" to use stack
+        # like peeking mode which "pop".
+        task-peeking-mode = "FIFO"
+
+
+        dedicated-thread-pool{ #settings for Helios.DedicatedThreadPool
             thread-count = 3 #number of threads
```

Comment:
We might need to adjust this too.... The dedicated threadpool sized might interfere with these settings on the `ForkJoinExecutor`